src/server/game/Spells/Spell.cpp

```diff
@@ -7284,6 +7287,13 @@ bool Spell::UpdatePointers()
             m_originalCaster = nullptr;
     }
 
+    if (m_focusObjectGUID)
+    {
+        focusObject = ObjectAccessor::GetGameObject(*m_caster, m_focusObjectGUID);
+        if (focusObject && !focusObject->IsInWorld())
```

Comment:
IsInWorld check is unneccessary, ObjectAccessor will never hand out pointers to objects not-in-world (other than players with FindConnectedPlayer)

Reply:
thanks, updated